jwst/datamodels/schemas/image.schema.yaml

```diff
@@ -37,46 +37,18 @@ allOf:
       fits_hdu: RELSENS2D
       default: 1.0
       datatype: float32
-    wavelength:
```

Comment:
You're sure we don't ever need the wavelength array with ImageModel? I'm thinking of simple spectroscopic cases like NIRISS SOSS and MIRI LRS, which use ImageModel. Do we only ever populate a WAVELENGTH array for NIRSpec fixed-slit and MSA data? If so, then I guess we don't need wavelength in ImageModel, because the NIRSpec cases use MultiSlitModel.

Reply:
Yes, as of now,  we only populate it for NIRSPEC spectroscopic data.